packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -95,11 +95,30 @@ export const OptimizeCiemssOperation: Operation = {
 			inProgressForecastId: '',
 			forecastRunId: '',
 			optimizationRunId: '',
-			modelConfigName: '',
-			modelConfigDesc: '',
 			optimizeErrorMessage: { name: '', value: '', traceback: '' },
 			simulateErrorMessage: { name: '', value: '', traceback: '' }
 		};
 		return init;
 	}
 };
+
+// Get the intervention output from a given optimization run
+export async function getOptimizedInterventions(optimizeRunId: string) {
```

Comment:
Im just correcting this now 
At the moment this is only working for 1 or the 2 types

Reply:
I don't understand what you are saying